addons/l10n_fi/data/account_tax_report_line.xml

```diff
@@ -10,15 +10,10 @@
         <field name="sequence">0</field>
         <field name="report_id" ref="vat_report"/>
     </record>
-    <record id="tax_report_sales_25_5" model="account.tax.report.line">
-        <field name="name">25.5 %:n vero</field>
-        <field name="code">sale_25_5</field>
-        <field name="tag_name">fi_320</field>
-        <field name="sequence">10</field>
-        <field name="report_id" ref="vat_report"/>
-        <field name="parent_id" ref="tax_report_sales_title"/>
-    </record>
```

Comment:
I don't believe the line will be back soon. There is only one line needed. It used to say 24% and reference 24%. Now it says 24% and refrences 24% and 25.5%. After the first of september until the end of the firscal year, it will say 25.5% and reference 24%. And after that it will say 25.5% and reference 25.5%. So this line won't be back, we'll just rename the 24% to 25.5%.